types/router.strict.ts

```diff
@@ -0,0 +1,53 @@
+import { Router } from 'worktop';
```

Comment:
This file best illustrates what a real-world usage looks like with the new type system. It's strongly/strictly type-safe, too!